### PR TITLE
March Webinar with Kudzai

### PR DESCRIPTION
**Pull request by gdziuba**
close https://github.com/FlowFuse/website/issues/1747

## Checklist

- [x]  image
- [x] doublecheck formId

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+date: 2024-03-26
+time: 17:00 CET (11:00am ET) 
+duration: 60
+# video: rl6uwV2PhOc
+hosts: ["grey-dziuba"]
+hubspot:
+    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
+---
+
+**The digital transformation journey for many manufacturers hasn’t always been a smooth one.  From new terminology, and technologies, to security concerns, progress is generally always hindered by one of these three. As manufacturing companies evolve their strategies from industry to industry, there is always one thing that is top of mind - trustworthy data; how to access it, how to share it, and how to keep it secure.**
```
Comment (Hasmin-AC) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:14]:
```suggestion
**What value does a UNS bring?  In this webinar, we will focus on why and how low-code plus UNS makes a powerful combination for citizen developers.**
```

File: src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<!--more-->
+
```
Comment (Hasmin-AC) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:17]: The digital transformation journey for many manufacturers hasn’t always been a smooth one.  From new terminology, and technologies, to security concerns, progress is generally always hindered by one of these three. As manufacturing companies evolve their strategies from industry to industry, there is always one thing that is top of mind - trustworthy data; how to access it, how to share it, and how to keep it secure.

Comment (Hasmin-AC) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:17]: The digital transformation journey for many manufacturers hasn’t always been a smooth one.  From new terminology, and technologies, to security concerns, progress is generally always hindered by one of these three. As manufacturing companies evolve their strategies from industry to industry, there is always one thing that is top of mind - trustworthy data; how to access it, how to share it, and how to keep it secure.

File: src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Building Unified Namespace using Node-RED and MQTT
+subtitle: Join Grey Dziuba, OT Data and Community Strategist, and Kudzai Manditereza, Developer Advocate at HiveMQ, as they discuss MQTT and Node-RED and how it pertains to the Unified Name Space, UNS.
```
Comment (Hasmin-AC) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:3]:
```suggestion
subtitle: Join Grey Dziuba, OT Data and Community Strategist, and Kudzai Manditereza, Developer Advocate at HiveMQ, as they discuss MQTT and Node-RED and how it pertains to the Unified Namespace, UNS.
```

File: src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+hosts: ["grey-dziuba"]
+hubspot:
+    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
+---
+
+**What value does a UNS bring?  In this webinar, we will focus on why and how low-code plus UNS makes a powerful combination for citizen developers.**
+
+<!--more-->
+
+Most industries function on a standard ISA-95, sometimes referred to as the Perdue model.  One of the biggest takeaways is making sure that you have an impenetrable firewall that sits between your manufacturing side and the enterprise networks.  Oftentimes leaving the manufacturing side, OT network, in the dark without access to the internet. With an environment like this, how is it possible to make data easily accessible and usable?
```
Comment (Hasmin-AC) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:18]:
```suggestion
The digital transformation journey for many manufacturers hasn’t always been a smooth one. From new terminology, and technologies, to security concerns, progress is generally always hindered by one of these three. As manufacturing companies evolve their strategies from industry to industry, there is always one thing that is top of mind - trustworthy data; how to access it, how to share it, and how to keep it secure.

Most industries function on a standard ISA-95, sometimes referred to as the Perdue model.  One of the biggest takeaways is making sure that you have an impenetrable firewall that sits between your manufacturing side and the enterprise networks.  Oftentimes leaving the manufacturing side, OT network, in the dark without access to the internet. With an environment like this, how is it possible to make data easily accessible and usable?
```

**Comment by gdziuba** [comment]
@Yndira-FlowForge,   The date for this is Tuesday, not Thursday.  Sorry, I should have called that out.  Can we adjust this?

**Pull request merged**